src/app/router/connectorRoutes.tsx

```diff
@@ -30,121 +33,239 @@ import raspiblitz from "/static/assets/icons/raspiblitz.png";
+    logo: citadel,
+  },
+  "lnd-hub-go": {
+    path: "lnd-hub-go",
+    element: <ConnectLndHub lndHubType="lndhub_go" />,
+    title: i18n.t("translation:choose_connector.lndhub_go.title"),
+    logo: lndhubGo,
+  },
+  "lnd-hub-bluewallet": {
+    path: "lnd-hub-bluewallet",
+    element: <ConnectLndHub />,
+    title: i18n.t("translation:choose_connector.lndhub_bluewallet.title"),
+    logo: lndhubBlueWallet,
+  },
+  eclair: {
+    path: "eclair",
+    element: <ConnectEclair />,
+    title: i18n.t("translation:choose_connector.eclair.title"),
+    logo: eclair,
+  },
+  kollider: {
+    path: "kollider",
+    element: <ConnectKollider variant="select" />,
+    title: i18n.t("translation:choose_connector.kollider.title"),
+    description: i18n.t("translation:choose_connector.kollider.description"),
+    logo: kolliderLogo,
+    children: kolliderConnectorRoutes,
+  },
+  [galoyPaths.bitcoinBeach]: {
+    path: galoyPaths.bitcoinBeach,
+    element: <ConnectGaloy instance={galoyPaths.bitcoinBeach} />,
+    title: i18n.t("translation:choose_connector.bitcoin_beach.title"),
+    logo: galoyBitcoinBeach,
+  },
+  [galoyPaths.bitcoinJungle]: {
+    path: galoyPaths.bitcoinJungle,
+    element: <ConnectGaloy instance={galoyPaths.bitcoinJungle} />,
+    title: i18n.t("translation:choose_connector.bitcoin_jungle.title"),
+    logo: galoyBitcoinJungle,
+  },
+  btcpay: {
+    path: "btcpay",
+    element: <ConnectBtcpay />,
+    title: i18n.t("translation:choose_connector.btcpay.title"),
+    logo: btcpay,
+  },
+};
+
+const umbrelConnectorRoutes: Route[] = [
+  connectorMap["umbrel-lnd"],
+  connectorMap["lnc"],
+  connectorMap["commando"],
+  connectorMap["lnbits"],
+];
+
+const citadelConnectorRoutes: Route[] = [
+  connectorMap["citadel"],
+  connectorMap["lnc"],
+  connectorMap["commando"],
+  connectorMap["lnbits"],
+];
+
+const raspiblitzConnectorRoutes: Route[] = [
+  connectorMap["raspiblitz-lnd"],
+  connectorMap["lnc"],
+  connectorMap["commando"],
+  connectorMap["lnbits"],
+];
+
+const myNodeConnectorRoutes: Route[] = [
+  connectorMap["mynode-lnd"],
+  connectorMap["lnc"],
+  connectorMap["commando"],
+  connectorMap["lnbits"],
+];
+
+const start9ConnectorRoutes: Route[] = [
+  connectorMap["start9-lnd"],
+  connectorMap["lnc"],
+  connectorMap["commando"],
+  connectorMap["lnbits"],
+];
+
+function getDistribution(key: string, children: Route[]): ConnectorRoute {
+  const name = i18n.t(normalizeKey(`translation:distributions.${key}.name`));
+
+  return {
+    path: key,
+    element: (
+      <ChooseConnector
+        title={i18n.t("translation:distributions.title", { name })}
+        description={i18n.t("translation:distributions.description", { name })}
+        connectorRoutes={children}
+      />
+    ),
+    title: i18n.t(normalizeKey(`translation:choose_connector.${key}.title`)),
+    logo: distributionMap[key]["logo"],
+    children,
+  };
+}
+
+const distributionMap: { [key: string]: { logo: string } } = {
```

Comment:
It seems strange here you only store the logo - you could pass the logo directly to the getDistribution function, right? or alternatively you could also store the children in this map and change getDistribution to have a single argument?

Reply:
I refactored to the latter.